### ISSUE TITLE
CLI: Add `--all` option to automated install commands

### ISSUE THREAD

**Issue by mbercx** [aiidateam/aiida-pseudo#101]
Since some users might simply want install all pseudopotential configurations of a certain family (I'm running into this when installing pseudos for the [QEapp](https://github.com/aiidalab/aiidalab-qe) for example), it would be good to add an `--all` options that simply installs all configurations.

**Comment by mbercx** [comment]
Note: Perhaps there should be an argument here to specify the version though. Typically you don't want to install all versions.

**Comment by mbercx** [comment]
Also pinging @csadorf and @yakutovicha so they are aware of this idea for a future `aiidalab-docker-stack` release.

**Comment by sphuber** [comment]
Think indeed not including all versions might be a good idea. Especially for PseudoDojo this can lead to *a lot* of families. Thinking about it, is this always well defined? Many of the versions in PD do not contain all types. For example v1.0 only has the jthxml pseudos no?

**Comment by mbercx** [comment]
> Think indeed not including all versions might be a good idea. Especially for PseudoDojo this can lead to _a lot_ of families. Thinking about it, is this always well defined? Many of the versions in PD do not contain all types. For example v1.0 only has the jthxml pseudos no?

Yeah, I'm considering for the automated install of all pseudos in a family with multiple formats, we should probably also make sure that they can only install one format. I'm not sure about the jthxml pseudos for v1.0, would have to check. 😅
